microprofile.ls/com.redhat.microprofile.ls/src/main/java/com/redhat/microprofile/ls/JavaTextDocumentService.java

```diff
@@ -102,9 +102,12 @@ public void didClose(DidCloseTextDocumentParams params) {
 
 	@Override
 	public void didSave(DidSaveTextDocumentParams params) {
-		triggerValidationFor(documents.all().stream().map(TextDocument::getUri).collect(Collectors.toList()));
+		// validate all opened java files which belong to a MicroProfile project
+		triggerValidationForAll(null);

```

Comment:
we're losing the info about which file was saved, it'll certainly come in handy later

Reply:
> we're losing the info about which file was saved, it'll certainly come in handy later

We don't need this information, the idea is when a java file is saved, all diagnostics are recomputed (to managed diagnostics for Java files which could depend on the saved java file).

You want I add the uri as parameter in triggerValidationForAll (even if it is not used)?